setup.py

```diff
@@ -9,7 +9,7 @@
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
 requirements = [
-    'requests >= 2.5.2',
+    'requests >= 2.5.2, < 2.11',
```

Comment:
hmm, #1204 just removes the pin. so it's actually ok to use requests-2.11 without any other code changes?